packages/fxa-settings/src/pages/Legal/Privacy/index.test.tsx

```diff
@@ -40,13 +40,15 @@ describe('Legal/Privacy', () => {
     jest.clearAllMocks();
   });
 
-  it('renders as expected', () => {
+  it('renders as expected', async () => {
     renderWithLocalizationProvider(<LegalPrivacy />);
-    testAllL10n(screen, bundle);
+    await waitFor(() => {
+      testAllL10n(screen, bundle);
```

Comment:
Just a word of caution - I've ocasionally seen false positives with grouped assertions in a `waitFor` block and/or non-explicit assertions (vs using `expect(...`).